code/modules/research/departmental_lathe.dm

```diff
@@ -19,6 +19,7 @@
 	allowed_department_flags = DEPARTMENTAL_FLAG_ALL|DEPARTMENTAL_FLAG_ENGINEERING
 	department_tag = "Engineering"
 	circuit = /obj/item/circuitboard/machine/protolathe/department/engineering
+	console_link = TRUE

```

Comment:
uh ok setting a single var is not going to help you here.

Reply:
My testing > your conjecture

It wouldn't link, I changed the var, it linked.

Reply:
Have you tested whether everything displays correctly)

Reply:
please answer me robustin